sources/api/netdog/src/wicked/mod.rs

```diff
@@ -206,6 +206,12 @@ impl WickedInterface {
         }
     }
 
+    /// Add config to accept IPv6 router advertisements
+    // TODO: expose a network config option for this
+    pub(crate) fn accept_ra(&mut self) {
+        self.ipv6 = Some(WickedIpv6::default())
+    }
```

Comment:
I would've expected this change, and the changes to `write_config_file` to be in the previous commit.

Reply:
Good catch.... That was how I intended it to be; must have mixed it up somewhere along the way.  Will fix.